packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -336,12 +337,37 @@ export async function checkAndInstall(): Promise<Result<void, FxError>> {
   );
 }
 
-// TODO: use _checkAndInstall for new prerequisite task
-// export async function checkAndInstallV2(
-//   orderedCheckers: PrerequisiteOrderedChecker[],
-//   ports: number[]
-// ): Promise<Result<void, FxError>> {
-// }
+export async function checkAndInstallV2(
```

Comment:
Better naming than v2...
- `checkAndInstallForTask`
- or, `checkAndInstallHandler`
- or, renaming the old one to `checkAndInstallLegacy`

Reply:
Updated it to `checkAndInstallForTask`, thx.